### PR TITLE
deps: update modeling dependencies

### PR DESCRIPTION
**Pull request by smbea**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,7 +41,7 @@ ___Note:__ Yet to be released changes appear here._
 ## 1.5.0
 
 * `FEAT`: allow external variable providers to be added ([variable-resolver#1](https://github.com/bpmn-io/variable-resolver/pull/1))
-* `DEPS`: update to `bpmn-js@1.19.1`
+* `DEPS`: update to `bpmn-js@11.5.1`
```
Comment (smbea) [inline CHANGELOG.md:44]: Typo in previous changelog update

[the remark this answers is not in the document]
Comment (smbea) [reply CHANGELOG.md:44]: Yeah this was a small typo I noticed. Likely came from copying the `bpmn-js-properties-panel` version bellow

**Comment by marstamm** [comment]
Looks good to me, note that @nikku will cut a patch release for `bpmn-js-properties-panel` shortly. Cf. https://github.com/bpmn-io/bpmn-js-properties-panel/pull/902

**Comment by smbea** [comment]
Fixed conflicts and updated changelog version already with https://github.com/camunda/camunda-bpmn-js/pull/267/commits/dc636a8dd310c019d1d0c4bdd13ee8878b1cf48d

**Review by marstamm — APPROVED** [review]
🚀

**Pull request merged**